apps/src/templates/CountryAutocompleteDropdown.jsx

```diff
@@ -28,9 +28,14 @@ export default class CountryAutocompleteDropdown extends Component {
     showErrorMsg: PropTypes.bool,
     showRequiredIndicator: PropTypes.bool,
     value: PropTypes.string,
+    fieldName: PropTypes.string,
```

Comment:
Should fieldName be required?

Reply:
It has a default value below. My impression of this (from the PR description) is that this prop is just for overriding the default when needed.

Reply:
Yes, this is optional. The default is the value that was previously hard coded, for backward compatibility.